### DISCUSSION TITLE
Saving a trained model

### DISCUSSION THREAD

**Discussion by danilmezor** [edtechre/pybroker#134] [General]
Hi everyone,

Is there a way to save a trained model(s)? 
I been trying to find a solution but it seems like ModelTrainer uses custom wrappers and I cant access models functions like predict or anything else. Even if I try to pass trained models into training function and use it later, it gives me an error. 

Thanks!

**Comment by edtechre** [comment]
Hi @danilmezor,

Have you tried using the pretrained and predict_fn arguments to the [model()](https://www.pybroker.com/en/latest/reference/pybroker.model.html#pybroker.model.model) function?
